_data/scrapy.yml

```diff
@@ -1,10 +1,9 @@
 stable:
-  version: 1.6
+  version: 1.7.2
```

Comment:
I think it should be `1.7`, not `1.7.2`

Reply:
At the end we need to revise what are they used for:

```bash
$ ack '\.version' -a
_includes/download-button.html:4:    <p> <i class="fa fa-cloud-download "></i> Scrapy {{ stable.version }} </p>
index.html:131:        <li><a href="http://docs.scrapy.org/en/{{ stable.version }}/intro/overview.html">- Discover Scrapy at a glance</a></li>
download.html:26:			<a href="https://github.com/scrapy/scrapy/archive/{{ devel.version }}.tar.gz"><button>Tarball</button></a>
download.html:27:			<a href="https://github.com/scrapy/scrapy/archive/{{ devel.version }}.zip"><button>Zip</button></a>
download.html:39:		<a href="https://github.com/scrapy/scrapy/archive/{{ oldstable.version }}.zip">
download.html:40:		<h2 class="float">Looking for an old release? <br /> Download Scrapy {{ oldstable.version }}</h2>
```

Reply:
Of those links, I think only the documentation one should not be using `1.7.2`, and it should be using `stable.rtd` instead of `stable.version`.

Reply:
`1.7` would work as well, as Github would provide a download for the `1.7` branch. But I believe what we actually want downloaded is the tag of a specific release.

Reply:
I agree. good catch.